scripts/watcher/pstree.remy.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "pstree.remy" {
```

Comment:
Why not move this into a `types` folder under `scripts/watcher`?

Reply:
Sure